### ISSUE TITLE
Alpha values in TextArea background

### ISSUE THREAD

**Issue by seaeagle1** [bash0/cewe2pdf#116]
Just to share some of the hacking I've been doing for myself. I'm a fan of using semi-transparent backgrounds on my text boxes to make text more readable. However cewe2pdf ignores the alpha values in the backgroundcolor attribute. So, I've managed to figure out how to read and render it properly. See 687fe5052e4851aad2e2be26e0f66ea490f6494a.

I'm now trying to get the bounding box right, as I've got some text overrunning the background box now.

**Comment by AnEnglishmanInNorway** [comment]
Nice, thank you. I just used semi-transparent text background for the first time, so your fix was great to have!
I wonder if you ever made any progress on the text bounding box issue. I worked on that for a while without finding a 100% solution - the best I could manage was the warning message that today's code issues when it thinks there is an issue. I suspect that it might be something to do with font mismatches, but I don't really believe that either!
